src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -138,10 +159,52 @@ protected void SendKeys(params object[] keys)
             => Editor.SendKeys(keys);
 
         protected void DisableSuggestionMode()
-            => VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(false);
+        {
+            VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(false);
+            WaitForAsyncOperations(FeatureAttribute.Workspace);
+        }
 
         protected void EnableSuggestionMode()
-            => VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(true);
+        {
+            VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(true);
+            WaitForAsyncOperations(FeatureAttribute.Workspace);
+        }
+
+        protected void EnableQuickInfo()
```

Comment:
Should we just have the methods being Set* that take a Boolean rather than doubling all the methods here? This class's concept count is getting disturbingly big, disturbingly quick.